source/includes/_platform-payments.md

```diff
@@ -158,198 +153,179 @@ Updates a specific set of stored account details. This will trigger an AML check
 | sortCode      | string | The sort code.      | required |
 | accountName   | string | The account name.   | required |
 
+## *Registering Companies as Wallet Holders*
 
-## `POST /platformApi/wallet/company/registration`
+## `POST /companies 🚧`
 ```http
-POST /platformApi/wallet/company/registration HTTP/1.1
+POST /companies HTTP/1.1
```

Comment:
I don't know the answer to this, but will you need an _applicant_ from Modulr's perspective? Or are you just going to choose one from the list of partners / directors / etc.

Reply:
We were not sure on this.  @hoto can you advise?

If we do need to supply an applicant, will the applicant be from the list of persons of significant control?

We did speak about having another endpoint with which to supply additional information, but given we're not sure what that information is, we thought we'd defer creating anything until we've gone live.  Which might mean we create an endpoint, and expose it to workflows only.  Then publish it when we're more confident of the API.

Thoughts @edslocombe or @Tom-Goji ?